frontend/public/components/factory/list-page.jsx

```diff
@@ -33,13 +33,14 @@ export const TextFilter = (props) => {
     placeholder = `Filter ${label}...`,
     autoFocus = false,
     parentClassName,
+    ...otherInputProps
   } = props;
```

Comment:
Since we don't use this `props` var anywhere else, would it be possible to directly destructure the variables in the function argument itself?

Reply:
Even though it makes sense to destructure the props above, I wouldn't want to stop the PR because of that. I think it's okay to have it here since the change is trying to fix something else.